frontend/src/pages/Profile.jsx

```diff
@@ -1,7 +1,59 @@
-import React from 'react'
+import React from 'react';
+import "../components/Styles/Profile.css"
+import axios from "axios"
+import { useState } from 'react';
 
 export const Profile = () => {
+  
```

Comment:
inani a para limpyu tan awon
const [account, setAccount] = useState({
 username: "",
 //rest of the inputs...
})